EVA-1946 — Regroup Open Targets processing steps (#99)
EBIvariation/eva-cttv-pipeline: README.md

[![Build Status](https://travis-ci.com/EBIvariation/eva-cttv-pipeline.svg?branch=master)](https://travis-ci.com/EBIvariation/eva-cttv-pipeline)
[![Coverage Status](https://coveralls.io/repos/github/EBIvariation/eva-cttv-pipeline/badge.svg?branch=master)](https://coveralls.io/github/EBIvariation/eva-cttv-pipeline?branch=master)

# How to submit an Open Targets batch
Batch submission process consists of two major tasks, which are performed asynchronously:

> 1. [**Manual curation**](docs/manual-curation/README.md) of trait names should be performed approximately once every two months as new ClinVar versions come out, containing new trait names. The output of this step is used by the main evidence string generation pipeline.

Reviewer: ```suggestion
1. [**Manual curation**](docs/manual-curation/README.md) of trait names should be performed approximately once every two months as new ClinVar versions with new trait names are released. The output of this step is used by the main evidence string generation pipeline.
```

2. [**Evidence string generation**](docs/generate-evidence-strings.md) is mostly automated and should be run for every Open Targets batch submission.

Additional documentation:
* [Setting up the common environment](docs/environment.md) which is required by both protocols to be able to run

> * [Advanced build instructions](docs/build.md), which are not required for batch processing under normal circumstances

Reviewer: Could you add a gist as to when this **_is_** required?

Author: Sure, it sounds useful to clarify. Fixed in https://github.com/EBIvariation/eva-cttv-pipeline/pull/99/commits/cab489bb62d190a140b8fb0f13a554662ca8ad87
